### PR TITLE
Palm Release Notes

### PR DESCRIPTION
**Pull request by pdpinch**
Palm Release Notes. 

closes https://github.com/openedx/build-test-release-wg/issues/279

### Date Needed (optional)

Palm will be released on June 12 2023

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:.

- [ ] Subject matter expert: @sarina 
- [ ] Subject matter expert: @feanil 
- [ ] Subject matter expert: @kdmccormick 
- [ ] Subject matter expert: @ghassanmas
- [ ] Subject matter expert: @dianakhuang
- [ ] Subject matter expert: @pshiu
- [ ] Product review: @jmakowski1123 

fyi @blarghmatey @cachob @NotoriousMKD 

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [x] Build an RTD draft for your branch: https://docsopenedxorg--321.org.readthedocs.build/en/321/community/release_notes/palm.html

### Sandbox (optional)

- [x] https://palm.demo.overhang.io/

### Post-review

- [ ] Squash commits

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pdpinch!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by sarina** [comment]
Hey Peter - I'm on PTO thru the 12th. I'll review promptly after. Thanks
for working on this!

File: source/community/release_notes/palm.rst [hunk 1]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
 
 Learner Experiences
 *******************
 
+New Learner Home Page 
+=====================
+
+The new Learner Home has many of the same features of the old dashboard, with some extended functionality and 
+performance enhancements.
+
```
Comment (ghassanmas) [inline source/community/release_notes/palm.rst:61]: As per our discussion, I think learner-home/learner-dasboard MFE as flagged as expiremental, also it has been suggested by @arbrandes to create a tutor plugin for that. That being said I guess you need to a refrence to that plugin to refer here?

Comment (arbrandes) [reply source/community/release_notes/palm.rst:61]: @ghassanmas, I think so, yes.  If we're mentioning it in the release notes, it would be better if it lived either in the openedx or overhang orgs.

File: source/community/release_notes/palm.rst [hunk 2]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
+<https://openedx.atlassian.net/wiki/spaces/AC/pages/3617849345/Ecommerce+Deprecation+and+Replacement+Project>`_.
+
+
+Developer changes
+=================
+
+The old developer documentation site has been deprecated and removed. Please use `Open edX Developers - Latest 
+Documentation <https://docs.openedx.org/en/latest/developers/index.html>`_  instead.
+
+part of the Tutor API. This change should not affect most developers, who only use the :code:`Actions` and :code:`Filters`
```
Comment (sarina) [inline source/community/release_notes/palm.rst:43]: The beginning of this sentence appears to have been lost 😲

File: source/community/release_notes/palm.rst [hunk 3]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
+The old developer documentation site has been deprecated and removed. Please use `Open edX Developers - Latest 
+Documentation <https://docs.openedx.org/en/latest/developers/index.html>`_  instead.
+
+part of the Tutor API. This change should not affect most developers, who only use the :code:`Actions` and :code:`Filters` 
+classes (notice the plural) from :code:`tutor.hooks`. If it does affect you, there are upgrade instructions in the 
+discussion post, `Simplifying & documenting the Tutor Hooks API 
+<https://discuss.openedx.org/t/simplifying-documenting-the-tutor-hooks-api/9258>`_
+
+Most of the viewing code for PDF certificates was removed in previous releases. In the Palm release, we closed out 
+this work with an ADR (Application Decision Record) on `Leaving PDF Certificate Fields in Certificates Model
```
Comment (sarina) [inline source/community/release_notes/palm.rst:49]:
```suggestion
this work with an ADR (Architecture Decision Record) on `Leaving PDF Certificate Fields in Certificates Model 
```

Comment (sarina) [reply source/community/release_notes/palm.rst:49]: I'm not super convinced linking this ADR is that important. What information are you hoping to convey by including it?

File: source/community/release_notes/palm.rst [hunk 4]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
+<https://github.com/openedx/edx-platform/blob/open-release/palm.master/lms/djangoapps/certificates/docs/decisions/008-certificate-model-remnants.rst>`_. 
+
 
 Learner Experiences
 *******************
 
+New Learner Home Page 
+=====================
+
+The new Learner Home has many of the same features of the old dashboard, with some extended functionality and
```
Comment (sarina) [inline source/community/release_notes/palm.rst:59]:
```suggestion
The new Learner Home has many of the same features of the old learner dashboard, with some extended functionality and 
```

File: source/community/release_notes/palm.rst (hunk 1 above)
Comment (sarina) [reply source/community/release_notes/palm.rst:61]: If this is experimental, it should be moved to an "Experimental" part of the release notes as we've done in previous releases

File: source/community/release_notes/palm.rst [hunk 5]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
 Learner Experiences
 *******************
 
+New Learner Home Page 
+=====================
+
+The new Learner Home has many of the same features of the old dashboard, with some extended functionality and 
+performance enhancements.
+
+* The Learner Home is now built with Paragon, the open edX design pattern library. It is accessible and easy to style with brand colors.
```
Comment (sarina) [inline source/community/release_notes/palm.rst:62]:
```suggestion
* The Learner Home is now built with Paragon, the Open edX design pattern library. It is accessible and easy to style with brand colors. 
```

File: source/community/release_notes/palm.rst [hunk 6]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
 *******************
 
+New Learner Home Page 
+=====================
+
+The new Learner Home has many of the same features of the old dashboard, with some extended functionality and 
+performance enhancements.
+
+* The Learner Home is now built with Paragon, the open edX design pattern library. It is accessible and easy to style with brand colors. 
+* Course cards show the course thumbnail, information about the course, and the ability to upgrade to a paid track or view/begin a course (if applicable). Further course actions (e.g. unenroll, email opt-out settings, and social media share) have been moved to the menu/triple dot icon on the course card.
```
Comment (sarina) [inline source/community/release_notes/palm.rst:63]: Can you add an image of the new course card here? I find the image of the Refine pop up confusing without seeing the new course card to begin with.

File: source/community/release_notes/palm.rst [hunk 7]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
+.. image:: /_images/community/release_notes/palm/new_learner_home_filtering.png
+    :alt: The Refine pop-up with Course State and Sort options
+
+* All of a user’s courses are fetched on page load. To make the page manageable, we paginate that list, showing 25 courses at a time. To view other courses, a user should page through their list of courses using the pagination controls at the bottom of the course list.
+
+.. image:: /_images/community/release_notes/palm/new_learner_home_pagination.png
+    :alt: Next, previous and page number buttons appear below the list of courses
+
+* Site staff can now masquerade as users on the platform by typing a username or email in the “View as” box and hitting submit. This is designed to be “view only” so change actions (e.g. enroll, unenroll, selecting a session) are blocked in this view.
+* There is a complete list of changes in the `open edX wiki <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2022/11/21/3584589831/2U+New+Learner+Home+Page#Comparison-with-old-experience>`_.
```
Comment (sarina) [inline source/community/release_notes/palm.rst:75]:
```suggestion
* There is a complete list of changes in the `Open edX wiki <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2022/11/21/3584589831/2U+New+Learner+Home+Page#Comparison-with-old-experience>`_. 
```

File: source/community/release_notes/palm.rst [hunk 8]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
+rollout, 100 represents rollout to all users.
+
 
 Instructor Experiences
 **********************
 
+New Visual Problem Editor
+=========================
+
+The release includes an improved problem-authoring experience with an interactive WYSIWYG editor. Writing markdown
```
Comment (sarina) [inline source/community/release_notes/palm.rst:89]: I'm not sure if WSIWYG is too technical a term for these release notes? Maybe just "interactive editor"?

File: source/community/release_notes/palm.rst [hunk 9]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
 **********************
 
+New Visual Problem Editor
+=========================
+
+The release includes an improved problem-authoring experience with an interactive WYSIWYG editor. Writing markdown 
+code is no longer necessary. But, the advanced mode is still available, maintaining the ability to write and edit 
+OLX XML.
+
+The open edX wiki page `[2U] New Visual Problem Editor <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2023/04/07/3724312593/2U+New+Visual+Problem+Editor>`_
```
Comment (sarina) [inline source/community/release_notes/palm.rst:93]:
```suggestion
The Open edX wiki page `[2U] New Visual Problem Editor <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2023/04/07/3724312593/2U+New+Visual+Problem+Editor>`_ 
```

File: source/community/release_notes/palm.rst [hunk 10]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
+The release includes an improved problem-authoring experience with an interactive WYSIWYG editor. Writing markdown 
+code is no longer necessary. But, the advanced mode is still available, maintaining the ability to write and edit 
+OLX XML.
+
+The open edX wiki page `[2U] New Visual Problem Editor <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2023/04/07/3724312593/2U+New+Visual+Problem+Editor>`_ 
+provides a brief explanation of what has changed in the problem editor. Updated detailed instructions on writing 
+problems can be found in section `8.4. Working with Problem Components <https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_components/create_problem.html#working-with-problem-components>`_ of the Building and Running an edX Course documentation.
+
+The visual problem editor is an update to the Course Authoring Micro-frontend first released in Olive. Assuming that 
+you are already using the Course Authoring MFE, all you need to do to enable the Visual Problem Editor is add the
```
Comment (sarina) [inline source/community/release_notes/palm.rst:98]: How would I know if I was using the Course Authoring MFE? Was that an optional add in Olive? Is it not default/required in palm?

File: source/community/release_notes/palm.rst [hunk 11]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
+New ORA Grading Experience
+==========================
+
+In this new on-platform grading experience one can easily preview common file types, assign rubric values, provide 
+comments, and coordinate grading with all members of the course teams. Complete documentation is in section 
+`10.26.4. Staff Grading for Open Response Assignments 
+<https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/open_response_assessments/ORA_Staff_Grading.html#staff-grading-for-open-response-assignments>`_  of the Building and Running an edX Course documentation. 
+
+The new ORA grading experience depends on the ORA Grading Micro-frontend, which was included as an experimental 
+feature in Olive. To turn on the feature, add the feature flag :code:`openresponseassessment.enhanced_staff_grader`.
```
Comment (sarina) [inline source/community/release_notes/palm.rst:110]: To turn on the feature they first need to enable the mfe?

File: source/community/release_notes/palm.rst [hunk 12]
```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
...
+
+New Bulk Email Experience
+=========================
+
+Enabling the Communications Micro-frontend allows instructors to schedule bulk emails, instead of sending them
```
Comment (sarina) [inline source/community/release_notes/palm.rst:115]: Is this an experimental MFE in Palm? or is it on by default?

**Review by sarina — COMMENTED** [review]
Looking good so far!

File: source/community/release_notes/palm.rst (hunk 1 above)
Comment (ghassanmas) [reply source/community/release_notes/palm.rst:61]: The plugin is there https://github.com/ghassanmas/tutor-contrib-mfe-dashboard, let me know how/if we need to move it. I have test it with tutor/tutor-mfe palm branch. and it currenlty to my fork until https://github.com/openedx/frontend-app-learner-dashboard/pull/136 is resolved

File: source/community/release_notes/palm.rst (hunk 12 above)
Comment (arbrandes) [reply source/community/release_notes/palm.rst:115]: It is expected to be "on" by default, in the sense that it is usable without jumping through any hoops other than enabling the bulk email feature in the first place.  (This is a pre-existing flag that has been off since its implementation, years ago.)

File: source/community/release_notes/palm.rst [hunk 13]
```diff
@@ -15,22 +15,160 @@ These are the release notes for the Palm release, the 16th community release of
 Breaking Changes
 ****************
 
+Tutor minimum versions of Docker and Compose
+============================================
+
+In Palm the minimum required versions will be `Docker v20.10.15 <https://docs.docker.com/engine/release-notes/20.10/#201015>`_ and Compose v2.0.0. These versions date back from May 2022, so not everyone might be able to upgrade. We'll do our best to keep backward compatibility with non-buildkit installations, but users will not have access to the fanciest features, such as build-time cache.
```
Comment (pdpinch) [inline source/community/release_notes/palm.rst:21]: @regisb is this what you had in mind?

File: source/community/release_notes/palm.rst (hunk 3 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst:55]: I think it's important to resolve the deprecation that started a couple of releases ago, and reassure folks that the legacy certificate data won't be lost, even if the functional code is removed. 

Maybe this belongs in the deprecation section, where it will be less prominent?

File: source/community/release_notes/palm.rst (hunk 8 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst:69]: I thought WYSIWYG was a common term, but you're not the first person to question that recently.

File: source/community/release_notes/palm.rst (hunk 11 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst:90]: Yes

File: source/community/release_notes/palm.rst (hunk 10 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst]: Looking back at the Olive release notes, I see that the Course Authoring MFE was enabled by default, so I clarified this.

File: source/community/release_notes/palm.rst (hunk 12 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst]: Ok. I didn't realize that the MFE settings are handled by the Tutor plugin: 

- https://github.com/overhangio/tutor-mfe/pull/126/files#diff-e7cc1a222153171e9e8c772d71621d1394f40264ecb772a82212c23953a01fefR12
- https://github.com/overhangio/tutor-mfe/pull/126/files#diff-9dd8a655b8597bfd6cdb64b96b2a805174561cae462059559fdc4a2b8be0ac06R51

> other than enabling the bulk email feature in the first place

@arbrandes what setting are you refering to?

**Comment by UsamaSadiq** [comment]
Hi @pdpinch 
I have recently merged https://github.com/openedx/devstack/pull/1082 (Upgraded Mongo to 4.4 in devstack) in `palm.master` branch which is needed to be added in the palm release notes.

File: source/community/release_notes/palm.rst (hunk 13 above)
Comment (regisb) [reply source/community/release_notes/palm.rst:21]: Yes, pretty much. Thanks!

File: source/community/release_notes/palm.rst [hunk 14]
```diff
@@ -15,6 +15,46 @@ These are the release notes for the Palm release, the 16th community release of
...
+
+RateXBlock unsupported
+======================
+
+The RateXBlock has been deprecated and is now officially unsupported. `FeedbackXBlock 
+<https://github.com/pmitros/FeedbackXBlock>`_ is the recommended replacement.
```
Comment (sarina) [inline source/community/release_notes/palm.rst:27]: Is this an officially supported block? This isn't in the Open edX org

File: source/community/release_notes/palm.rst [hunk 15]
```diff
@@ -23,14 +63,109 @@ Learner Experiences
...
+<https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/open_response_assessments/ORA_Staff_Grading.html#staff-grading-for-open-response-assignments>`_  of the Building and Running an edX Course documentation. 
+
+The new ORA grading experience depends on the ORA Grading Micro-frontend, which was included as an experimental 
+feature in Olive. To turn on the feature, add the feature flag :code:`openresponseassessment.enhanced_staff_grader`.
+
+New Bulk Email Experience
+=========================
+
+The new bulk email experience for instructors is enabled automatically in Palm. It allows instructors to schedule bulk 
+emails, instead of sending them immediately.
```
Comment (sarina) [inline source/community/release_notes/palm.rst:96]: Is there more information on this? Screenshots maybe? How could I test it?

File: source/community/release_notes/palm.rst [hunk 16]
```diff
@@ -15,30 +15,220 @@ These are the release notes for the Palm release, the 16th community release of
...
+===========
+
+* Filter by active courses in the course listing API.
+* Added :code:`certificate_available_date` to course detail API.
+
+New filters
+===========
+
+* Added :code:`VerticalBlockRenderCompleted` filter that will be called after a VerticalBlock's rendering is completed.
+* Added :code:`CourseEnrollmentSiteFilterRequested` filter which receives enrollments information that can be used to for filtering which enrollments are returned to the user.
```
Comment (mariajgrimaldi) [inline source/community/release_notes/palm.rst:224]: This hasn't been included yet on edx-platform: https://github.com/openedx/edx-platform/pull/31331 so it's not available out of the box

File: source/community/release_notes/palm.rst [hunk 17]
```diff
@@ -15,30 +15,220 @@ These are the release notes for the Palm release, the 16th community release of
...
+API changes
+===========
+
+* Filter by active courses in the course listing API.
+* Added :code:`certificate_available_date` to course detail API.
+
+New filters
+===========
+
+* Added :code:`VerticalBlockRenderCompleted` filter that will be called after a VerticalBlock's rendering is completed.
```
Comment (mariajgrimaldi) [inline source/community/release_notes/palm.rst:223]: Also this one that was added in the same PR as `VerticalBlockRenderCompleted`: https://github.com/openedx/edx-platform/pull/31388/files#diff-33588454a586bfbea6bd37f51652b19fbb4e77785791c9de43eaa68c7b527788R120-R128
```suggestion
* Added :code:`VerticalBlockRenderCompleted` filter that will be called after a VerticalBlock's rendering is completed.
* Added :code:`VerticalBlockChildRenderStarted` that will be called when every child block of a VericalBlock is about to be rendered.
```

File: source/community/release_notes/palm.rst [hunk 18]
```diff
@@ -15,30 +15,220 @@ These are the release notes for the Palm release, the 16th community release of
...
+
+iOS App updates
+===============
+
+* Learners will see a course dates shift banner in the course header when the server returns that dates needs shifting.  
+
+Other New Features
+==================
+
+* Support for Persian language (locale code :code:`fa_IR`) with 100% translate and reviewd
```
Comment (sarina) [inline source/community/release_notes/palm.rst:85]:
```suggestion
* Support for Persian language (locale code :code:`fa_IR`) with 100% translate and reviewed
```

File: source/community/release_notes/palm.rst [hunk 19]
```diff
@@ -15,30 +15,220 @@ These are the release notes for the Palm release, the 16th community release of
...
+===================
+
+* The Learner dashboard was updated. Profile was moved from the header to the footer. Learners can choose between viewing My Courses and My Programs. 
+* The course header and course navigation have been updated
+* Learners will see a course dates shift banner in the course header when the server returns that dates needs shifting.  
+
+iOS App updates
+===============
+
+* Learners will see a course dates shift banner in the course header when the server returns that dates needs shifting.
```
Comment (sarina) [inline source/community/release_notes/palm.rst:80]: I have no idea what this means, could you explain?

File: source/community/release_notes/palm.rst (hunk 15 above)
Comment (sarina) [reply source/community/release_notes/palm.rst:123]: @pdpinch I put in some info about the bulk email experience in the "smol" blog post: https://docs.google.com/document/d/1IUIlsJX4EtmIBuJdbcBj-LQsGHn8wQyLkByiYj5Sips/edit#

File: source/community/release_notes/palm.rst (hunk 12 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst]: Adolfo pointed me to the settings that were documented back in the Eucalyptus release, https://docs.openedx.org/en/latest/community/release_notes/eucalyptus.html#updates-to-bulk-email -- I don't think we need to mention them here.

File: source/community/release_notes/palm.rst (hunk 14 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst:27]: The repo was moved to openedx-unsupported as part of the deprecation process: https://github.com/openedx-unsupported/RateXBlock

File: source/community/release_notes/palm.rst (hunk 15 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst]: I added a screenshot and also tried to explain the situation with scheduled sending, which is not easy to set up.

File: source/community/release_notes/palm.rst (hunk 19 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst:80]: Does the screenshot help?

File: source/community/release_notes/palm.rst (hunk 3 above)
Comment (sarina) [reply source/community/release_notes/palm.rst]: I think moving to the deprecation section is a great idea.

File: source/community/release_notes/palm.rst (hunk 19 above)
Comment (sarina) [reply source/community/release_notes/palm.rst:80]: I don't think I see a screenshot from the iOS app

File: source/community/release_notes/palm.rst [hunk 20]
```diff
@@ -15,30 +15,232 @@ These are the release notes for the Palm release, the 16th community release of
...
+=========================
+
+The new bulk email experience for instructors is enabled automatically for any courses that are configured for sending bulk emails. See the Eucalyptus release notes for instructions on configuring bulk email. 
+
+The instructor editing experience is largely the same, with choices of who to send the email to, and places to author the subject and body of the email:
+
+.. image:: /_images/community/release_notes/palm/communications_send_an_email.png
+    :alt: Send and Email interface for sending bulk emails 
+
+The new experience also allows an experimental feature for instructors to schedule bulk emails, rather than sending them immediately. This feature requires a cron service which isn't currently included in Tutor. For more details see `Processing Scheduled Instructor Tasks <https://github.com/openedx/edx-platform/blob/b74138f2e67d9c636286f4e4633316aed051d21f/lms/djangoapps/instructor_task/docs/decisions/002-processing-scheduled-instructor-tasks.rst#L21>`_
```
Comment (sarina) [inline source/community/release_notes/palm.rst:129]: 👍🏻

File: source/community/release_notes/palm.rst (hunk 19 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst]: whoops, forgot the `git push`

File: source/community/release_notes/palm.rst [hunk 21]
```diff
@@ -15,30 +15,241 @@ These are the release notes for the Palm release, the 16th community release of
...
+* Added :code:`VerticalBlockChildRenderStarted` and :code:`VerticalBlockRenderCompleted` filters that will be called at the start of rendering a vertical and after rendering is completed.
+* Added :code:`AccountSettingsRenderStarted` filter which passes the account settings context before is rendered.
+* Added events :code:`XBLOCK_PUBLISHED`, :code:`XBLOCK_DELETED`, and :code:`XBLOCK_DUPLICATED` to signal changes to xblocks in taxonomy-connector/course-discovery.
+
 
 Researcher & Data Experiences
 *****************************
 
+* Tracking events have been added for reporting of threads, responses and comments, and for marking responses as endorsed. 
+
```
Comment (pdpinch) [inline source/community/release_notes/palm.rst:250]: @bmtcril I found these news events that were added going through the palm commits. Is this accurate? Is it complete?

File: source/community/release_notes/palm.rst [hunk 22]
```diff
@@ -15,30 +15,237 @@ These are the release notes for the Palm release, the 16th community release of
...
+.. image:: /_images/community/release_notes/palm/new_learner_home_pagination.png
+    :alt: Next, previous and page number buttons appear below the list of courses
+
+* Site staff can now masquerade as users on the platform by typing a username or email in the “View as” box and hitting submit. This is designed to be “view only” so change actions (e.g. enroll, unenroll, selecting a session) are blocked in this view.
+* There is a complete list of changes in the `Open edX wiki <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2022/11/21/3584589831/2U+New+Learner+Home+Page#Comparison-with-old-experience>`_. 
+
+To turn on the New Learner home page, add and enable the :code:`learner_home_mfe.enabled` Waffle Flag in the django 
+admin. To support a phased role out, there is also a setting :code:`LEARNER_HOME_MFE_REDIRECT_PERCENTAGE`. Set it to an 
+integer value (0 - 100) representing what percentage of users should be redirected to Learner Home. 0 represents no 
+rollout, 100 represents rollout to all users.
```
Comment (arbrandes) [inline source/community/release_notes/palm.rst:172]: I suggest rewording this to point to the (hot-off-the-oven) experimental Tutor plugin installation instructions:

https://github.com/openedx/openedx-tutor-plugins/tree/main/plugins/tutor-contrib-learner-dashboard-mfe#learner-dashboard-mfe-plugin-for-tutor

File: source/community/release_notes/palm.rst (hunk 21 above)
Comment (bmtcril) [reply source/community/release_notes/palm.rst:246]: I wasn't aware of them, but I was speaking with @ayub02 about the existing events being maintained (they are). He might know more about the new events. We should document them here, too: https://docs.openedx.org/en/latest/developers/references/internal_data_formats/tracking_log_event_list.html

File: source/community/release_notes/palm.rst (hunk 22 above)
Comment (pdpinch) [reply source/community/release_notes/palm.rst]: Thanks. I replaced the instructions with your suggested link, and moved it up to the start of the section.

File: source/community/release_notes/palm.rst (hunk 21 above)
Comment (ayub02) [reply source/community/release_notes/palm.rst:241]: Hi! The new discussion related events can be found [here](https://docs.google.com/spreadsheets/d/1YoOWlodw6lutfI-8pDZDRIOqjo6J9D5L2SuENn-mXmQ/edit?usp=sharing). I'll be on vacation from 19th June till mid July and won't be able to update the docs, until after. Feel free to use the sheet to update the docs if you want.

Comment (bmtcril) [reply source/community/release_notes/palm.rst:241]: Hi @ayub02 I've taken an initial stab [here](https://github.com/openedx/docs.openedx.org/pull/343), but I'm not super clear on the spreadsheet meanings or how up to date it is (looks like there are things in the actual columns which are not matching what's expected). I haven't gone through every tab, but it looks like at least the fields on search also changed?

Is the intention that what is in the "actual" column matches what's in the code right now for every event on every tab? I can take a more thorough pass if that's the case.

**Review by sarina — APPROVED** [review]

File: source/community/release_notes/palm.rst [hunk 23]
```diff
@@ -15,30 +15,232 @@ These are the release notes for the Palm release, the 16th community release of
...
+* Site staff can now masquerade as users on the platform by typing a username or email in the “View as” box and hitting submit. This is designed to be “view only” so change actions (e.g. enroll, unenroll, selecting a session) are blocked in this view.
+* There is a complete list of changes in the `Open edX wiki <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2022/11/21/3584589831/2U+New+Learner+Home+Page#Comparison-with-old-experience>`_. 
+
+Other Operator changes
+======================
+
+* Mongo is updated to v4.4. 
+* Added a setting :code:`expiration_datetime_is_explicit` to the CourseMode model to override the default behavior of the Upgrade Deadline where it is set to be 10 days after the course start date. 
+* Operators can now use a shared ElasticSearch installation for multiple OpenedX instances, using the setting :code:`ELASTIC_SEARCH_INDEX_PREFIX`. 
+* the Micro-frontend (MFE) config setting `SUPPORT_URL` has been added to all MFEs. Use it to set the
```
Comment (Agrendalath) [inline source/community/release_notes/palm.rst:175]: @pdpinch, this part seems to be missing some information.

File: source/community/release_notes/palm.rst (hunk 21 above)
Comment (ayub02) [reply source/community/release_notes/palm.rst:241]: @bmtcril Correct. The "Actual" column is what's in the code (at least as per my testing). There are differences between actual and intended, but they don't need to be addressed on priority, in my opinion.
